mk/spksrc.compile.mk

```diff
@@ -36,7 +40,7 @@ endif
 pre_compile_target: compile_msg
 
 compile_target:  $(PRE_COMPILE_TARGET)
-	@$(RUN) $(PSTAT_TIME) $(MAKE) -j$(NCPUS) $(COMPILE_MAKE_OPTIONS)
+	@$(RUN) $(PSTAT_TIME) $(MAKE) $(COMPILE_MAKE_OPTIONS)
```

Comment:
If the `-j` flag ends-up going back here (which I would personally prefer), the previous `ifeq ($(filter $(NCPUS),0 1),)` would be a really nice addition down here.

And again, if the `-j` comes back here then this would mean re-using the `-@MAKEFLAGS= $(MAKE) ...` I did for java-11-openjdk https://github.com/SynoCommunity/spksrc/pull/4859/commits/9d59a0c6b353471f0b4a99e41e5c6cfc9bbd645f and consider it as an exception.

Reply:
Thanks I think its better safe than sorry, so I'll do that. :+1:

Reply:
@th0ma7  If `-j` is back here and `$(NCPUS)` is filtered than https://github.com/SynoCommunity/spksrc/commit/9d59a0c6b353471f0b4a99e41e5c6cfc9bbd645f is not necessary. Since `-j` is not added with `DISABLE_PARALLEL_MAKE=1`.